src/_includes/partials/module_card.liquid

```diff
@@ -0,0 +1,42 @@
+<a
+  href="{{ href }}"
+  aria-labelledby="{{ title }}"
```

Comment:
If I'm not mistaken, the attribute `aria-labelledby` designates what other element within the DOM serves as label for this element. See: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby

Reply:
Yes. A bit further down I had assigned an `id` on the heading which would mean that this link is labelled by the module title in the `<h2>`. However, since the heading is also separated by `<span>`s etc. I'm not sure anymore if it works. I went for the safer bet with the `aria-label` now.

Reply:
You should prefer using `aria-labeledby` over `aria-label` but my issue here is that the `title` cannot be used as an `id` because it can contain spaces. You had the right intention, I just got confused because I didn't expect the `id` to be a valid one, but rather the same text you would place into a regular `arial-label`

Reply:
Okay, got it! 👍